$NetBSD: patch-configure,v 1.1 2022/12/05 10:17:48 jperkin Exp $

Remove ugly upstream hack that breaks the build on Darwin.

--- configure.orig	2022-11-17 16:16:51.000000000 +0000
+++ configure
@@ -12618,7 +12618,7 @@ else $as_nop
 fi
 
 			;;
-		*darwin*)
+		*notdarwin*)
 			# 23 October 2022: See README_d/README.macosx for
 			# the details on what's happening here. See also
 			# the manual.
